### PR TITLE
Add my hot chocolate story

### PR DESCRIPTION
**Pull request by luddles82**
This is my contribution to our story, hope its acceptable. Thanks Nick.

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/make-hot-chocolate/hot-choc.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+the cup of cocoa you find in your hand.
+
+You think that's a bit gross as he's essentially offering some
+of himself to you but you go with it.
+
+That drink turns out to be the best hot chocolate and marshmallows ever!
+
+Once you finish it you think what shall I do next?
+
+[Go for a run to burn off those sugary calories](create-your-own-adventure/english/run/run.md).
```
Comment (casey-collab) [inline english/make-hot-chocolate/hot-choc.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/make-hot-chocolate/hot-choc.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+[Do a spot of yoga](create-your-own-adventure/english/yoga/yoga.md).
```
Comment (casey-collab) [inline english/make-hot-chocolate/hot-choc.md]: This link is also invalid.

File: english/make-hot-chocolate/hot-choc.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+You have a word with the marshmallow who tried to grab the lighter.
```
Comment (casey-collab) [inline english/make-hot-chocolate/hot-choc.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/make-hot-chocolate/hot-choc.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+That drink turns out to be the best hot chocolate and marshmallows ever!
+
+Once you finish it you think what shall I do next?
+
+[Go for a run to burn off those sugary calories](create-your-own-adventure/english/run/run.md).
+
+[Do a spot of yoga](create-your-own-adventure/english/yoga/yoga.md).
+
+[Look for a way to leave](https://github.com/luddles82/create-your-own-adventure/blob/master/english/find-exit/leave.md)
```
Comment (cbuckey-uda) [inline english/make-hot-chocolate/hot-choc.md]: Please don't include an absolute to your own GitHub profile. Instead, it should be a relative link like your other ones.

File: english/make-hot-chocolate/hot-choc.md (hunk 2 above)
Comment (cbuckey-uda) [inline english/make-hot-chocolate/hot-choc.md]: This won't quite work, as it looks for the `create-your-own-adventure` directory within the current directory (`make-hot-chocolate`) and doesn't find it. You could use a link such as `../run/run.md`.

File: english/make-hot-chocolate/hot-choc.md (hunk 4 above)
Comment (luddles82) [inline english/make-hot-chocolate/hot-choc.md]: Ok thanks Caroline. As the file was a bit of a mess I decided to delete the branch and start again. I've now added a story about lego. Thanks for the course btw, it was really informative and I enjoyed it a lot. Nick.

Comment (cbuckey-uda) [inline english/make-hot-chocolate/hot-choc.md]: Okay, glad you enjoyed the course!

**Pull request closed without merging**